packages/web/src/shared/components/form/RichTextInput.tsx

```diff
@@ -66,7 +66,9 @@ const RichTextInput = forwardRef<RichTextInputRef, RichTextInputProps>(
     });
 
     useEffect(() => {
-      editor?.commands.setContent(value ?? '', false);
+      editor?.commands.setContent(value ?? '', false, {
+        preserveWhitespace: 'full',
```

Comment:
This was the issue with the whitespace. By default tiptap and prosemirror let HTML condense the whitespace like normal